# L16-20 Gene regulation in eukaryotes

## Chromatin

+ H1, H2A, H2B, H3 and H4 -> histone proteins
+ positively charged (rich in basic aa e.g. lysine and arginine, 20 to 24% of sequence) so negative phosphates of DNA attach
+ neutralise DNA so allows tighter packing
+ histones interact mainly with the minor groove of DNA -> H bond to oxygen and OH of AT regions and backbone
+ have a stem loop structure at the ends to avoid degradation (instead of poly A tail)
+ in S phase when histones are needed (dna replication) stem loop is stabilised, in rest of the cycle it is unwound to degrade the histones
+ example of stability control
+ highly conserved across evolution since necessary to eukaryotic gene regulation

**nucleosome** = unit of chromatin, equal to DNA of 200bp wrapping around a histone complex of: 2 molecules of H2A, H2B, H3 and H4. one molecule of H1 sits above the nucleosome and provides tighter packing. -> H1 present in half the amount of the other four. Role is to compact DNA and produce another level of regulation between euchromatin and heterochromatin. More specifically, two H2A-H2B dimers and one H3-H4 tetramer. Histone 2AZ variant replaced H2A in some nucleosomes —> suggested that proximity between H2AZ containing nucleosome and the transcription start site is an indication of gene expression i.e. if H2AZ close to TSS the gene has a higher expression level! Nucleosome displaced from TSS to being transcription, and blocks the TSS in inactive genes. So H2AZ may destabilise the nucleosome and also may recruit RNA polymerase II.
> R. Bargaje 2012 “Proximity of H2A.Z containing nucleosome to the transcription start site influences gene expression levels in the mammalian liver and brain”

In a partial digest use enzymes that only cut the lengths of DNA between the nucleosomes (i.e. the exposed DNA), showing them to be about 200bp long -> “*beads on a string*” model [halt DNA digest before all the dna is digested to nucleotides, hence partial]

Beads on a strong are not the tightest form possible. H1, on the nucleosome, allows tighter packing than this into a 30nm fibre -> could either be a single helix (one start model, solenoid structure) or a helical ribbon (two start model, made via zig zagging the DNA.)

(histone octamer: x2 H2A, H2B, H3 and H4 and 146bp of DNA wrapped around the histones to shorted in, plus 54bp that serve as a “linker” to link  nucleosomes together -> histone H1 associated with nucleosome, but once linker is digested histone H1 is released so histone H1 does further packing of the beads on a string into a 30nm fibre)

After 30nm fibre -> looped to get a 300nm structure, heterochromatin [euchromatin goes into a the heterochromatin structure in mitosis and meiosis since chromosomes must go to the right place, easier to segregate if they’re tightly packed]

## Structure & expression of eukaryotic genes

Typical eukaryotic gene: box is transcribed into RNA after the initiation site, first base transcription is +1,

RNA polymerase III = tRNA transcription and other RNA level molecules, don't need to be converted into protein<br/>
RNA polymerase II = mRNA from protein coding genes<br/>
RNA polymerase I = ribosomal RNAs<br/>

Function was found by using alpha amanitin to inhibit RNA polymerases -> II inhibited first, so mRNA production lost, then III knocked out to lose transcription of tRNA and others.
