# L16-20 Gene regulation in eukaryotes

## Chromatin

+ H1, H2A, H2B, H3 and H4 -> histone proteins
+ positively charged (rich in basic aa e.g. lysine and arginine, 20 to 24% of sequence) so negative phosphates of DNA attach
+ neutralise DNA so allows tighter packing
+ histones interact mainly with the minor groove of DNA -> H bond to oxygen and OH of AT regions and backbone
+ have a stem loop structure at the ends to avoid degradation (instead of poly A tail)
+ in S phase when histones are needed (dna replication) stem loop is stabilised, in rest of the cycle it is unwound to degrade the histones
+ example of stability control
+ highly conserved across evolution since necessary to eukaryotic gene regulation

**nucleosome** = unit of chromatin, equal to DNA of 200bp wrapping around a histone complex of: 2 molecules of H2A, H2B, H3 and H4. one molecule of H1 sits above the nucleosome and provides tighter packing. -> H1 present in half the amount of the other four. Role is to compact DNA and produce another level of regulation between euchromatin and heterochromatin. More specifically, two H2A-H2B dimers and one H3-H4 tetramer. Histone 2AZ variant replaced H2A in some nucleosomes —> suggested that proximity between H2AZ containing nucleosome and the transcription start site is an indication of gene expression i.e. if H2AZ close to TSS the gene has a higher expression level! Nucleosome displaced from TSS to being transcription, and blocks the TSS in inactive genes. So H2AZ may destabilise the nucleosome and also may recruit RNA polymerase II.
> R. Bargaje 2012 “Proximity of H2A.Z containing nucleosome to the transcription start site influences gene expression levels in the mammalian liver and brain”

In a partial digest use enzymes that only cut the lengths of DNA between the nucleosomes (i.e. the exposed DNA), showing them to be about 200bp long -> “*beads on a string*” model [halt DNA digest before all the dna is digested to nucleotides, hence partial]

Beads on a strong are not the tightest form possible. H1, on the nucleosome, allows tighter packing than this into a 30nm fibre -> could either be a single helix (one start model, solenoid structure) or a helical ribbon (two start model, made via zig zagging the DNA.)

(histone octamer: x2 H2A, H2B, H3 and H4 and 146bp of DNA wrapped around the histones to shorted in, plus 54bp that serve as a “linker” to link  nucleosomes together -> histone H1 associated with nucleosome, but once linker is digested histone H1 is released so histone H1 does further packing of the beads on a string into a 30nm fibre)

After 30nm fibre -> looped to get a 300nm structure, heterochromatin [euchromatin goes into a the heterochromatin structure in mitosis and meiosis since chromosomes must go to the right place, easier to segregate if they’re tightly packed]

## Structure & expression of eukaryotic genes

Typical eukaryotic gene: box is transcribed into RNA after the initiation site, first base transcription is +1,

RNA polymerase III = tRNA transcription and other RNA level molecules, don't need to be converted into protein<br/>
RNA polymerase II = mRNA from protein coding genes<br/>
RNA polymerase I = ribosomal RNAs<br/>

Function was found by using alpha amanitin to inhibit RNA polymerases -> II inhibited first, so mRNA production lost, then III knocked out to lose transcription of tRNA and others.

### RNA polymerase II:

Promoter = TATA box at -30 upstream, found in most eukaryotic genes, defines the start site (if you move TATA you move start site, TATA box defines start site 30 base pairs downstream from it’s location) & transcription factor binding sites e.g. SP1 -> delete TTA box to stop transcription to show that it’s an initiation factor.

TATA box = non-coding DNA sequence (cis-regulatory element)

TATA box recognised by TFIID and TFIIA “recognition event” —> (transcription factor 2 stands for RNA polymerase 2)
then TF2B also binds —> RNA polymerase 2 recognises and binds to TF2D,A&B —> TF2F associated with RNA polymerase and also binds —> TF2H binds and phosphorylates the polymerase (a kinase), so transcription may begin —> TF2D and A are left on the DNA for the next transcription event

Note: RNA polymerases never recognise DNA sequences, but recognise transcription factors bound to the DNA!

^”*basal transcriptional complex*” slow rate hence “basal” so needs transcription factors

summary: TF2D & A —> TF2B —> RNA polymerase & TF2F —> TF2H —> all dissociate but TF2D&A

#### Enhancer
= interacts with the promoter to increase transcription rate, distance not an issue, can be anywhere on the gene, in any orientation: action is independent of distance, orientation and position. Binding site of a transcription factor.
e.g. SP40 eukaryotic virus has:
+ CAT box (binding site for the RNA transcription factor)
+ SP1 binding site (SP1 = zinc finger transcription factor that binds to GC-rich motifs)  
+ 5000bp so easier to study, has all the eukaryotic transcription control sequences

Enhancer action can open the chromatin to allow the TF2D to bind more easily. How does an enhancer effect the promotor? Idea could be “tracking” - a protein binds the enhancer and then searches downstream until it finds the promoter, or “looping”, where the enhancer and promoter are close together in three-dimensional space.

Note: in humans many promoters contain CpG islands (i.e. cystine residue on the CpG dimer is methylated) -> act as sites of transcription initiation (70% of human genes, all housekeeping ones), usually wont coincide with a TATA box, will allow transcription start over a range of sites (not as focused.) BUT e.g. -globin gene has a TATA box and a CpG island.
> A. Deaton CpG islands and the regulation of transcription 2011

### RNA polymerase I:

Transcribes one gene for ribosomes, which is repeated in tandem in the DNA with spacers in between which are not transcribed. Makes 28S, 18S and 5.8S ribosome units = 45S precursor made then cleaved. No splicing in humans, need constant ribosome expression so don’t need control.

1. upstream binding factor (UBF) binds the rRNA gene promotor -50 upstream
2. SL1 factor allows RNA polymerase 1 recruitment —> transcription occurs

self splicing occurs in protozoa (tetrahymena) not humans -> intron in the primary rRNA transcript splices itself out, no need for protein assembly so can be done in vitro with just ion solutions -> Nobel prize for organic solution self splicing. All splicing info is held within the RNA.  [origin of splicing!]   

### RNA polymerase III:
5S of ribosome and tRNA

5S ribosomal RNA, genes are tandemly repeated in the DNA, has internal promoters —> TF3A and TF3C bind to the site at +50, TF3B binds at the +1 site, RNA polymerase III is recruited. Found by removing the upstream sequence and transcription still occured. Deletion of downstream also continued to allow transcription.

as 5S concentration goes up TF3A starts to bind to the mRNA sequence as well as the DNA sequence -> removes TF3A from DNA so acts as a negative feedback to lower 5S concentration.

### mRNA assembly:

**Addition of 5’ cap**: a methylated guanosine is added to the first transcribed base via a 5’-5’ interaction [i.e. the phosphate on the C5 of the nucleotide is bound to the c5 of the guanosine.] 5 prime to 5 prime bond so different to other phosphodiester bonds. The N7 position is methylated. Post transcriptional modification. Not encoded by DNA, added de novo. Ribosome binds to the 5’cap (guide feature) and initiates translation at the AUG codon. Polio virus doesn’t cap their mRNA -> protein binds to end of the RNA, allows it to be translated, so a cap mimicking protein + polio destroys the capping system of the host so only virus RNA translated.

**Poly A tail**: mRNA cleaved about 20-30bp downstream of the AAUAAA site = poly A site. Poly A site is preserved (we cut after it) and then 200 A residues are added for stability (protects from degradation like a telomere.) GU rich sequence is cleaved.

CPSF factor binds to the AAUAAA region, and CstF binds to the G/U sequence downstream of the poly adenylation site.

![polyA](http://www.imgt.org/IMGTeducation/Aide-memoire/_UK/poly/polyadenylation.png)

The complexes bind together to fold the mRNA over, exposing the poly A site for cleavage. The sequence with CstF is degraded, whilst the main mRNA sequence with the CPSF has a poly A tail added by poly A polymerase. 3’ ends usually vulnerable to exonuclease function, protected by tail. Not occurring on rRNA or tRNA so good way to select protein coding mRNA by using t affinity chromatography. Histones don’t have poly A tails -> have a stem loop structure instead via base pairing. When histone protein replication occurs (in S phase) stem loop structure is present so histones made from the mRNA, but in other phases the stem loop doesnt form -> no RNA made so no translation as we don’t need -> example of translational control.

> nuclear export of mRNAs: mRNA with bound proteins = messenger ribonucleoprotein; hnRNP C tetramer measures transcript length, Tap-p15 mediates mRNA translocation by associating with FG repeats on nucleoporins in the nuclear envelope. “Nuclear Export of Messenger RNA” 2015 J. Katahira.

**Splicing**: must remove the introns from mRNA (non coding DNA present in the gene and in the primary transcript of mRNA.) Once mRNA leaves the nucleus it must be fully processed i.e. only exons. Exon exit the nucleus, introns stay.

GU —-> intron <—— AG

Thus 5’ splice site = GU, 3’ splice site = AG

5’ splice site is cut, leaving exon 1 and intron&exon2.

Exon 1 could react back to remake the previous structure, as it is held by the splisosome in place, but a lariat is formed by the intron: the 5’-UG reacts with an A residue (branch point) in the intron via a 5’-2’ bond (since the 3’-OH is being used in the DNA backbone.)5’ G cannot now back-react. Once the system has defined the branch point, the next AG  (3’ splice site) is cut to release exon 2.

![splice](https://upload.wikimedia.org/wikipedia/commons/thumb/2/24/RNA_splicing_reaction.svg/400px-RNA_splicing_reaction.svg.png)

Guiding molecules: splisosome has *URNAs* (rich in uridine) to catalyse splicing progress.

Exon 1 held in place by the splisosome organelle so cannot diffuse away. [Autoimmune disease SLE -> antibodies made against their own proteins to inhibit splicing. Used to study splicing reaction.] SM proteins are associated with URNAs.

U1 binds to the 5’ splice site at GU, and U2 binds to the A residue in the intron, aka the branch point. U5 binds to exon 1. U4/6 replaces U1 at the UG 5’ splice site, resulting in U6 and U2 interaction -> this bends the DNA and allows the lariat to form. Now the 5’ splice site can be cut —-> exon 1 released, lariat and exon two remain —> exon 2 released by 3’ splicing —-> intron with associate proteins is free

All proteins left on the lariate to leave the RNA “clean” for translation.

![urnas](https://www.mpibpc.mpg.de/289655/original-1335965016.jpg?t=eyJ3aWR0aCI6ODAwLCJoZWlnaHQiOjYwMCwib2JqX2lkIjoyODk2NTV9--29621efe587f6d59729cfc050243fec63dc53e55)
