### Translation

#### Main principles and key players of protein translation

mRNA ---> amino acids in polypeptide chain  </br>

transcriptome in humans = 100,000 mRNAS </br>
Proteome in humans = 400,000 proteins </br>

this discrepancy is due to alternative splicing, where one mRNA precursor molecule can make multiple different proteins from one single gene by splicing them differently and arranging the exons in new ways. [polycistronic mRNAs are known to exist in eukaryotic viruses: Ryabova LA,  Pooggin MM,  Hohn T. Viral strategies of translation initiation: ribosomal shunt and reinitiation, Prog Nucleic Acid Res Mol Biol , 2002, vol. 72 (pg. 1-39)& in drosophila but not yet found in humans.]

Gene expression is governed by extracellular signals which result in a protein being transcribed and translated. 

![iet](https://image.slidesharecdn.com/7-150517143650-lva1-app6891/95/73-translation-8-638.jpg?cb=1441510763)

**mRNAs**: carrier of information in translation; remember uses U not T!; 64 possible codons, 3 of which are stop codons (UAA, UGA, UAG); Methionine is the common start codon AUG -> defines the beginning of the *Kozac sequence* in eukaryotes. *Shine-Dalgarno*, the prokaryotic sequence, is a ribosome binding site that occurs 8 bases upstream of the AUG codon. 

All amino acids apart from methionine and tryptophan have multiple codons coding for them = degenerate genetic code.

> Kozac sequence: (gcc)gccRccAUGG

> Shine-Dalgarno (SD) Sequence is a ribosomal binding site in bacterial and archaeal messenger RNA, generally located around 8 bases upstream of the start codon AUG: AGGAGG.

#### Map of the mRNA pre translation:
5'- 7-methyl guanosine cap ---> 5'-UTR ---> coding region from start AUG codon to stop codon --> 3' UTR--> polyA tail -'3

The guanosine is not coded for by the DNA but **added on after transcription**, it is methylated on the 7 position directly after capping in vivo by a methyltransferase. Capping protects from degradation, allows functionality. 

The polyA tail allows the mRNA to *circularise*, which is needed in translation. It allows Rho independent termination to halt transcription in prokaryotes and helps maintain RNA stability.

tRNAS: 75-95 bases long, about 100 types in eukaryotes --> all have a CCA 3' terminal sequence on the acceptor stem where amino acids get added. The loop opposite the 3' and 5' ends is the anticodon loop, which gives the tRNA its specificity to bind to a specific codon.  

the tRNA+amino acid structure = aminoacyl-tRNA

aminoacyl tRNA synthetase =
	+ enzyme that matches the amino acid with its correct tRNA molecule
	+ 20, one for each amino acid
	+ forms an aminoacyl-tRNA
	+ amino acid can now be added to the peptide via aminoacyl tRNA transferase
	+ enzyme features an editing site to ensure that the right tRNA has been added

aminoacyl tRNA transferases = enzymes that take the amino acid from the aminoacyl-tRNA and adds it to the polypeptide chain

These enzymes must be able to distinguish between tRNAs -> via the acceptor arm, which has stem and 3'end regions specific to each tRNA. The discriminator base proceeding the 3' CCA end (position 73) is particularly useful in identification of the right tRNA! 

![trna](https://qph.fs.quoracdn.net/main-qimg-bb031494c6d8ee8563df53d08760d6fb-c)

![closeup](https://res.mdpi.com/def502000344e1f7922be9c232011a933c481675f43518352b06bfa9956defef3cbb7dcdad088703d0a54fb50b035438d97e7d086eaaf28eace54ebf166be247b3919a04a684b977d3cddd342a5f23a2902f5e3d867ec5315800f708b69c521ec8f0735dca00d6eb7e96a935fdb690168c9a6081508f61fc3c6b78627e8bc85e1111dd066a161f6ac63b87686b6c8d18c8de8648eb951034)

As there are less than 61 tRNAs there is not one tRNA for each codon, so non-Watson Crick base pairing often occurs between the codon and the anticodon. Results in wobble between the 3' base of the mRNA and the 5' base of the tRNA anticodon.

![wobble](https://www.mun.ca/biology/scarr/MGA2-03-30.jpg)

#### **Ribosomes**: protein synthesising machines in the cytoplasm 
	+ able to elongate polypeptide chains at a rate of 3-5 amino acids per second
	+ ribosomes are structural molecules lacking enzymes but have "ribozymes" in the large subunit with catalytic functions
	+ ribosomes have both RNA and protein in their structure
	+ ribosomes move along the mRNA during translation
	+ 80S complex is made up of subunits
	+ ribosomes are only made when there is a demand for protein synthesis, otherwise they ar degraded
	+ ribosomes are made in the nucleolus and moved out into the cytoplasm  
	+ ribosomes hold the mRNA and the tRNAs together during translation:

mRNA binds to the small subunit, the P site in the ribosome interacts with the tRNA carrying the growing polypeptide chain, the A site interacts with the tRNA carrying the next amino acid to be added. 

![epasite](http://education.med.nyu.edu/mbm/proteinSynthesisBasics/images/ribosome3BindingSites.png)

![model](http://emboj.embopress.org/sites/default/files/highwire/embojnl/36/7/854/embed/graphic-1.gif)

Translation initiated differently in prokaryotes and eukaryotes! 

Prokaryotes = Shine Dalgarno/Ribosome Binding Site -> found 5-9 bases upstream of the start AUG codon, there is where the 16S subunit of the ribosome will bind to the mRNA. 

Eukaryotes = Kozac sequence -> 5’CCACCAUGG 3’ found around the start methionine. This is not the ribosome binding site however, that occurs either in the 5' cap or as an internal ribosome entry site (IRES.) 

[In a few bacterial mRNAs, GUG is used as the initiator codon, and CUG occasionally is used as an initiator codon for methionine in eukaryotes & some mitochondria read UGA, the stop codon, as a Trp codon p133 lodish.]

**Polysomes:** complex of an mRNA strand where multiple ribosomes are able to translate the strand at the same time. mRNA is circularised here to allow this process. This allows many copies of the protein to be made from one mRNA strand. 
 
The fate of newly translated proteins:
	+ translocation, via a leader sequence informing the cell of the protein destination (e.g. transmembrane proteins shuttled to the plasma membrane via exocytosis pathways, see protein trafficking lectures.) 
	+ post translational modification in ER and Golgi -> this includes disulphide bridges in the ER!!!!

+ activation e.g. phosphorylation in Golgi: creates a dock in the molecule -> works in signalling pathways!
	     + Free OH needed (serine, threonine, tyrosine), turned into the PO4 by protein kinase
	     + e.g. lipid kinase can phosphorylate things on the membrane (where it sits)
